synapse/lib/httpapi.py

```diff
@@ -322,6 +322,32 @@ async def get(self):
             self.write(json.dumps(mesg))
             await self.flush()
 
+class ReqValidStormV1(Handler):
+
+    async def post(self):
```

Comment:
could use some test coverage improvements in this guy

Reply:
we had a few handlers doing the same two steps over and over again so i refactored that down into a function